templates/includes/sidebar.html

```diff
@@ -0,0 +1,11 @@
+<ul class="sidebar-nav list-group">
+  {% if user.is_authenticated %}
+    <li class="list-group-item">User: <a href="{{ user.get_absolute_url }}">{{ user.get_username }}</a></li>
```

Comment:
The method `get_username` should be called with parentheses as `user.get_username()`. Consider updating this to ensure it functions correctly.